package.json

```diff
@@ -59,7 +59,7 @@
     "typescript": "^5.3.0"
   },
   "volta": {
-    "node": "18.19.0",
-    "yarn": "4.1.1"
+    "node": "20.17.0",
```

Comment:
Not sure why CI is failing. Is this node version too new?

Reply:
Trying to figure it out...